plugins/node-cert/commands/cert-info.md

```
@@ -0,0 +1,71 @@
+# node-cert cert-info
+
+Display details about a certificate file.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)